src/components/Layout/Header/NavBar/WalletButton.tsx

```diff
@@ -30,17 +29,13 @@ type WalletImageProps = {
   isConnected: Boolean
 } & Pick<InitialState, 'walletInfo'>
 
-const WalletImage: React.FC<WalletImageProps> = ({ isConnected, walletInfo }) =>
-  isConnected ? (
-    <Image
-      boxSize='24px'
-      loading='lazy'
-      showBorder={false}
-      objectFit='contain'
-      bg='transparent'
-      src={walletInfo?.icon}
-    />
-  ) : null
+const WalletImage: React.FC<WalletImageProps> = ({ isConnected, walletInfo }) => {
```

Comment:
Nitpick: We shouldn't be using React.FC. this component doesn't render children. Arg typedefs are better for these type of components

Reply:
youre nitpick is correct @technojak will fix